examples/github-deploy-only.yml

```
@@ -0,0 +1,81 @@
+name: Build and upload to PyPI
+
+on: [push, pull_request, release]
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      name: Install Python
+      with:
+        python-version: '3.7'
+
+    - name: Install cibuildwheel
+      run: |
+        python -m pip install cibuildwheel==1.4.2
+
+    - name: Install Visual C++ for Python 2.7
+      if: startsWith(matrix.os, 'windows')
+      run: |
+        choco install vcpython27 -f -y
+
+    - name: Build wheels
+      run: |
+        python -m cibuildwheel --output-dir wheelhouse
+
+    - uses: actions/upload-artifact@v2
+      with:
+        path: ./wheelhouse
```

```suggestion
        path: ./wheelhouse/*.whl
```

You *might* need this to make sure it doesn't put the wheelhouse folder in the artifact, and to make sure these properly get merged. (I don't think so, but I've only tested it with the `*`)

---

Done as well. I guess it can't hurt, anyway. Let's stay as careful as possible, as these examples are not tested...